app/src/main/res/values-ar/strings.xml

```diff
@@ -63,6 +63,9 @@
     <string name="add_new_checklist_items">إضافة عناصر قائمة تدقيق جديدة</string>
     <string name="checklist_is_empty">قائمة التدقيق فارغة</string>
     <string name="remove_done_items">Remove done items</string>
+    <string name="sort_direction">Sort direction</string>
+    <string name="sort_separate_done_from_undone">Separate done items from undone items</string>
+    <string name="sort_checklist">Sort checklist</string>
```

Comment:
we dont need this many new strings. Let's reuse "Sort by" from commons, also dont use any string for "Sort direction". Just add a divider line there, like in other apps. You can check the file manager too, there are 2 dividers used at the sorting dialog.

Reply:
lets actually remove the whole "Separate done items from undone items" as there is already a "Move done checklist items at the bottom" in the app settings and their combination could become messy.